requirements.txt

```diff
@@ -1 +1,5 @@
-.
\ No newline at end of file
+.
+simppl~=1.0.4
```

Comment:
I'm not sure I understand this dependency - the tool depends on itself?

Reply:
hmm. looks like a mistake